src/authorizer/authorizer.lambda.ts

```diff
@@ -0,0 +1,34 @@
+
+export async function handler(event: any):Promise<any> {
+
+  // Step 1: Extract the token from the Authorization header
+  const token = event.authorizationToken;
+
+  // Log the received token for debugging purposes
+  console.log('Received Token:', token);
```

Comment:
In prod willen we deze niet loggen denk ik?

Reply:
Als niet in gebruik is deze comment overbodig

Reply:
niet in gebruik